app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -121,6 +125,35 @@ class AttendeeViewModel(
             })
     }
 
+    fun cancelPendingOrder() {
+        val order = mutablePendingOrder.value
+        val identifier: String? = orderIdentifier
+        if (order == null || identifier == null) return
+
+        order.status = ORDER_STATUS_CANCELLED
+        compositeDisposable += orderService.editOrder(identifier, order)
```

Comment:
Can't we use delete order here?

Reply:
I tried but there was error so I switchde to cancelling. Today, I have just taken a look at the code in the server and I think it forbid deleting by someone who is not organizer. Please take a look in here
<img src="https://i.ibb.co/f4cstMQ/fsf.png" alt="fsf" border="0">

Reply:
updated